README.md

```diff
@@ -24,37 +24,52 @@ python3 -m pip install --extra-index-url https://artifacts.luxonis.com/artifacto
  - cmake >= 3.4
  - C++14 compiler (clang, gcc, msvc, ...)
  - Python3
+ - Submodule dependencies with Git
```

Comment:
I would leave this out as its managed "automatically" (as part of the instructions) and doesn't require any additional user installed things.

Reply:
I have hit so many times the baffling situation where a build fails claiming stuff is missing. Perhaps added for myself only in the first place!

It feels like a RTM kind of situation, yet a friendly reminder…

We confirm the drop?

```suggestion
```

Reply:
That's why `git config --global submodule.recurse true` or just locally to specific repo without global option is useful, so it's done automatically, you don't have to. It should be default IMO in git.
Can be disabled on demand.

Reply:
Please confirm the deletion by committing the suggestion, if that’s the preference.

I do not know the full story why that is not the default. I like it manual when packaging with Git, so what happens remains under explicit control (to me, necessary for responsibly claim I know what’s in). Here it may be too much of a stretch in thinking :-)

This update is really about clarifying for new users like me. The line we discuss here is droppable, to me just a nice to have(tm).